app/services/metrc_service/plant/move.rb

```diff
@@ -60,12 +60,12 @@ def next_step(previous_growth_phase = nil, new_growth_phase = nil) # rubocop:dis
 
         new_growth_phase.downcase!
 
-        return :move_plant_batches if previous_growth_phase.include?('clone') && new_growth_phase.include?('clone')
-
-        return :move_plants if previous_growth_phase.include?('veg') && new_growth_phase.include?('veg')
+        return :move_plant_batches if %w[veg clone].any? { |phase| previous_growth_phase.include?(phase) && new_growth_phase.include?(phase) }
```

Comment:
not sure this does what we want. This condition evaluates to `false` when `previous_phase = 'veg'` and `next_phase = 'clone'` ?

I think the right code would be` %w[veg clone].include?(previous) && %w[veg clone].include?(next)`